ompi/datatype/ompi_datatype.h

```diff
@@ -188,20 +188,20 @@ ompi_datatype_add( ompi_datatype_t* pdtBase, const ompi_datatype_t* pdtAdd, size
 OMPI_DECLSPEC int32_t
 ompi_datatype_duplicate( const ompi_datatype_t* oldType, ompi_datatype_t** newType );
 
-OMPI_DECLSPEC int32_t ompi_datatype_create_contiguous( int count, const ompi_datatype_t* oldType, ompi_datatype_t** newType );
```

Comment:
These are more complicated as we would need to change the `ddt_elem_desc` and `ddt_loop_desc` structs to have `count` as int. Unfortunately, that will change the size of these structs, and increase the overall size of the datatype representation.